.nvmrc

```diff
@@ -0,0 +1 @@
+v4.2.1
```

Comment:
OK, I just rebased and rebuilt https://github.com/noirbizarre/sifter.js/tree/nested-properties-built until brianreavis/sifter.js#27 is merged. It should work on Mac too.